zhmccli/_cmd_ldap_server_definition.py

```diff
@@ -367,4 +367,4 @@ def cmd_ldapdef_delete(cmd_ctx, ldapdef_name):
 
     cmd_ctx.spinner.stop()
     click.echo("LDAP server definition '{lsd}' has been deleted.".
-               format(lsd=ldapdef_name))
+               format(lsd=ldapdef.properties['name']))  # Modified argument name
```

Comment:
This is one of the consequences of the reuse of the `ldapdef` name and would not be needed if you had not renamed it.